Gemfile

```diff
@@ -87,6 +87,7 @@ group :development do
   gem "spring"
   gem "spring-watcher-listen"
   gem "web-console"
+  gem "bullet"
```

Comment:
Can you add this in alphabetical order please? within the `development` group?

Reply:
done